{'project_id': '1211', 'year': '1989', 'x': 390270, 'y': 2498100, 'plot_id': '4213', 'strata': '41'}
Pandas(Index=0, project_id='1211', plot_id='4213', x=390270, y=2498100)
called process_plot
Pandas(Index=0, project_id='1211', plot_id='4213', x=390270, y=2498100)
4213
Directory '/efs/timesync/1989/audit/' already exists.
Data has been pickled and saved to '/efs/timesync/1989/audit/1989_4213_1211.p'.
............................Failed qa: 1989-02-02T16:28:39.7270380Z
..............Failed qa: 1989-02-18T16:28:47.137047Z
..............Failed qa: 1989-03-06T16:28:27.232528Z
.......Failed qa: 1989-03-22T16:27:34.7290750Z
.......Failed qa: 1989-04-07T16:27:58.080557Z
..............Failed qa: 1989-04-23T16:27:41.402031Z
..............Failed qa: 1989-05-09T16:27:28.848069Z
............................Failed qa: 1989-06-10T16:26:53.478072Z
.......Failed qa: 1989-06-26T16:26:34.002541Z
............................Failed qa: 1989-07-28T16:25:56.530547Z
.......Failed qa: 1989-08-13T16:25:33.858018Z
.......Failed qa: 1989-09-14T16:24:38.275575Z
..............Failed qa: 1989-09-30T16:24:19.5350000Z
..............Failed qa: 1989-10-16T16:24:06.6720880Z
.....................Failed qa: 1989-11-17T16:23:07.3560440Z
.......Data has been pickled and saved to '/efs/timesync/1989/audit/1989_4213_1211_qa_rejects.p'.
Success tracking goes here
